src/OptionalArgChecks.jl

```diff
@@ -3,17 +3,17 @@ module OptionalArgChecks
 using IRTools: @dynamo, IR, recurse!, block, branches, branch!
 using MacroTools: postwalk
 
```

Comment:
How about reexporting for convenience:
```suggestion
using ArgCheck: @argcheck, @check
export @argcheck, @check
```
```

Reply:
Yes, I'll do that. I also still need to add docs for that feature. I just didn't get to it yet